plugins/woocommerce/src/StoreApi/Routes/V1/AbstractTermsRoute.php

```diff
@@ -37,7 +37,8 @@ public function get_collection_params() {
 		$params['per_page'] = array(
 			'description'       => __( 'Maximum number of items to be returned in result set. Defaults to no limit if left blank.', 'woocommerce' ),
```

Comment:
Should this text be updated?
```suggestion
			'description'       => __( 'Maximum number of items to be returned in result set.', 'woocommerce' ),
```

Reply:
Yep!

I _am_ considering removing the `maximum` here and allowing for unbounded queries. It doesn't have a `default` configured so the expectation is currently that, unless explicitly desired, it will return all terms. There's no great way to give errors to notify the consumer that the query is no longer valid.

The attribute terms (options you can select for a given attribute) likely won't ever be so much that it breaks anything. Some folks have a lot of product categories, however, we're only talking about a tiny string and I doubt it'll be easy to exhaust memory with categories. I can see a lot of good use-cases for it too.

Reply:
Since I've decided to allow for these queries to remain unbounded, there's no longer a need to change the description.